### PR TITLE
Change 'VTX (MSP)' to 'VTX (MSP + Displayport)' and lose 'OSD (MSP Displayport)' option

### PR DESCRIPTION
**Pull request by SteveCEvans**
See also https://github.com/betaflight/betaflight/pull/11950

These two PR's remove the explicit MSP Displayport enablement, and instead have Displayport on MSP enabled at the same time at MSP VTX support if MSP is also enabled on the port. No traffic will flow on the serial port unless `osd_displayport_device = MSP`.

### CODE REVIEW

File: locales/ca/messages.json
Comment (haslinghuis) [inline locales/ca/messages.json:1523]: Please remove. We only need to fill up `locales/en` as changes to other locales will be overwritten.

File: locales/ca/messages.json
Comment (haslinghuis) [inline locales/ca/messages.json:1525]: Please remove. We only need to fill up `locales/en` as changes to other locales will be overwritten.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
